python/tk_multi_workfiles/my_tasks/my_tasks_model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, project, user, extra_display_fields, parent, bg_task_manager=
         """
         self.extra_display_fields = extra_display_fields or []
         filters = [["project", "is", project],
-                   ["task_assignees", "is", user]]
+                   ["task_assignees", "is", user],
```

Comment:
@pscadding Yes, my thoughts exactly, including the support for using the {context.user} notation. You should be able to refactor the functionality quite easily.